Client/game_sa/CFileLoaderSA.cpp

```diff
@@ -28,6 +28,11 @@ void CFileLoaderSA::StaticSetHooks()
     HookInstall(0x538690, (DWORD)CFileLoader_LoadObjectInstance, 5);
 }
 
+CEntitySAInterface* CFileLoaderSA::LoadObjectInstance(SFileObjectInstance* obj)
+{
+    return ((CEntitySAInterface * (__cdecl*)(SFileObjectInstance*))0x538090)(obj);
```

Comment:
This function (0x538090) is expecting a second `const char`* argument. See the function's prologue:
![modelName](https://github.com/multitheftauto/mtasa-blue/assets/12121551/22d1b817-a7c1-4df4-9314-a44b24c41f09)

Reply:
If fact, this argument is no actually used. So you can just rewrite it the following way: `return ((CEntitySAInterface * (__cdecl*)(SFileObjectInstance*, const char*))0x538090)(obj, nullptr);`

Reply:
Didn't notice that